src/utils/index.js

```diff
@@ -1,30 +1,73 @@
 import accepts from 'attr-accept'
 
+// Error codes
+export const FILE_INVALID_TYPE = 'file-invalid-type'
+export const FILE_TOO_LARGE = 'file-too-large'
+export const FILE_TOO_SMALL = 'file-too-small'
+export const TOO_MANY_FILES = 'too-many-files'
+
+// File Errors
+export const getInvalidTypeRejectionMsg = accept => {
```

Comment:
Sorry, I meant to say error here instead of message, as the return value(s) for these fns are actually `FileError`:
```suggestion
export const getInvalidTypeRejectionErr = accept => {
```